src/ast/mod.rs

```diff
@@ -873,12 +873,42 @@ impl fmt::Display for Assignment {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct FunctionNamedArg {
+    pub name: Ident,
+    pub arg: Expr,
+}
+
+impl fmt::Display for FunctionNamedArg {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{} => {}", self.name, self.arg)?;
+        Ok(())
+    }
+}
```

Comment:
Is there a reason to make this a separate struct, rather than `FunctionArg::Named { name, arg }`?

_(Another option is to make the argument type a simple struct with an optional `name`, and the reason I suggested `enum` initially because of the [other possible kinds of SQL arguments](https://jakewheat.github.io/sql-overview/sql-2016-foundation-grammar.html#SQL-argument)...)_

Reply:
Fixed , 
I agree that enum is better.